lib/project_types/script/commands/disable.rb

```diff
@@ -3,20 +3,14 @@
 module Script
   module Commands
     class Disable < ShopifyCli::Command
-      options do |parser, flags|
-        parser.on('--api_key=APIKEY') { |t| flags[:api_key] = t }
-        parser.on('--shop_domain=MYSHOPIFYDOMAIN') { |t| flags[:shop_domain] = t }
-      end
-
-      def call(args, _name)
-        form = Forms::Enable.ask(@ctx, args, options.flags)
-        return @ctx.puts(self.class.help) unless form
+      prerequisite_task :ensure_env
```

Comment:
Based on this [comment](https://github.com/Shopify/shopify-app-cli/issues/745#issuecomment-659423214) from Tim, we should use `use_dev_store`.